### DISCUSSION TITLE
Configuring storage with multiple time-series databases

### DISCUSSION THREAD

**Discussion by zhangj-923** [ccfos/nightingale#1247] [Q&A]
配置了两个时序库存储，vm和influxdb，influxdb的页面显示404
页面信息
![image](https://user-images.githubusercontent.com/67679447/199165650-70124ecf-a7c1-4afc-9e80-50dfb7b956e1.png)
配置信息
![image](https://user-images.githubusercontent.com/67679447/199165724-c05ac200-c3c7-4909-96e1-f0ced06e2002.png)
日志输出信息
![image](https://user-images.githubusercontent.com/67679447/199165817-b91941ac-09da-462b-82e9-21fe0b067671.png)

**Comment by UlricQin** [comment]
不支持influxdb，如果要用influxdb，得在前面搞一个prometheus，把influxdb作为prometheus的backend，再把prometheus作为夜莺的cluster

**Reply by zhangj-923** [reply]
好的 了解了
